languagetool-core/src/test/java/org/languagetool/GlobalConfigTest.java

```diff
@@ -0,0 +1,62 @@
+package org.languagetool;
```

Comment:
Could you add the standard license header we use? (With your name as the copyright holder?)